protoc-gen-go/generator/generator.go

```diff
@@ -1136,6 +1120,10 @@ func (g *Generator) printAtom(v interface{}) {
 		fmt.Fprint(g, v)
 	case *float64:
 		fmt.Fprint(g, *v)
+	case PackageName:
+		g.WriteString(string(v))
+	case ImportPath:
+		g.WriteString(string(v))
```

Comment:
Shouldn't this be `g.WriteString(strconv.Quote(string(v)))`?
That way you don't need to keep calling strconv.Quote manually below.

Reply:
Good thought. Done.